backend/src/controller/email.ts

```diff
@@ -25,7 +25,7 @@ Hello James,
 
 You have been invited to clean for Steve
```

Comment:
This needs to change! 😬

Reply:
Honestly don't even know how this got in here 🤦‍♂️